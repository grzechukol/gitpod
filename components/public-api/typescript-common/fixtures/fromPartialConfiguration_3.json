{
  "id": "123",
  "workspaceSettings": {
      "workspaceClass": "huge"
  },
  "prebuildSettings": {
      "enabled": true,
      "prebuildInterval": 5
  },
  "restrictionSettings": {
    "allowedWorkspaceClasses": ["huge", "large"]
  }
}
