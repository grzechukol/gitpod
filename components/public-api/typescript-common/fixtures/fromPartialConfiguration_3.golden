{
    "result": {
        "id": "123",
        "settings": {
            "prebuilds": {
                "enable": true,
                "prebuildInterval": 5
            },
            "workspaceClasses": {
                "regular": "huge"
            },
            "allowedWorkspaceClasses": [
                "huge",
                "large"
            ]
        }
    },
    "err": ""
}
